cmd/neofs-node/config.go

```diff
@@ -553,8 +557,8 @@ func initCfg(appCfg *config.Config) *cfg {
 		scriptHash:          contractsconfig.Netmap(appCfg),
 		state:               netState,
 		workerPool:          netmapWorkerPool,
-		needBootstrap:       !relayOnly,
-		reBoostrapTurnedOff: atomic.NewBool(relayOnly),
+		needBootstrap:       !atomicRelayOnly.Load(),
+		reBoostrapTurnedOff: atomicRelayOnly,
```

Comment:
suggest to drop pointer from this field, afaik it doesn't make any sense right now

Reply:
they [ask](https://cs.opensource.google/go/go/+/refs/tags/go1.21.1:src/sync/atomic/type.go;l=139) not to copy it, it wont pass `go vet` linter

Reply:
```go
c.internals.healthStatus.Store(int32(control.HealthStatus_HEALTH_STATUS_UNDEFINED))
c.cfgNetmap.reBoostrapTurnedOff.Store(relayOnly)
```
below

Reply:
NPE then? can you elaborate on your thoughts?

Reply:
or you meant to rely on the fact that `cfg` struct is used via the pointer always?

Reply:
We usually embed structures, not pointers if we don't copy the structure anyway (always accessing via pointer). And yes, this allows to skip init in many cases (like below).